test/test_automated_install.py

```diff
@@ -24,8 +24,8 @@ def test_setupVars_are_sourced_to_global_scope(Pihole):
         # Currently debug test function only
         echo "Outputting sourced variables"
         echo "PIHOLE_INTERFACE=\${PIHOLE_INTERFACE}"
-        echo "IPv4_ADDRESS=\${IPv4_ADDRESS}"
-        echo "IPv6_ADDRESS=\${IPv6_ADDRESS}"
+        echo "IPV4_ADDRESS=\${IPV4_ADDRESS}"
+        echo "IPV6_ADDRESS=\${IPV6_ADDRESS}"
         echo "PIHOLE_DNS1=\${PIHOLE_DNS1}"
```

Comment:
I'll pause here till I can do local tests and not mess up the pull. I know where that last problem is, and I'm looking at the code now.